icu4j/main/core/src/main/java/com/ibm/icu/text/PersonName.java

```diff
@@ -164,7 +164,7 @@ enum FieldModifier {
          * When used in conjunction with "initial", preserves the punctuation between the initials
          * as it existed in the original name field (e.g., "Jean-Claude" initializes into "J-C"
          * instead of "J.C.")  Has no effect if "initial" isn't also present in the list of modifiers.
-         * @draft ICU 74
+         * @internal
          */
         RETAIN("retain"),
```

Comment:
```suggestion
         * @internal
         * @deprecated This API is ICU internal only.
         */
         @Deprecated
        RETAIN("retain"),
```
as in
https://github.com/unicode-org/icu/blob/d501225db2bfed4fee3049827d6e9076dae2d1ad/icu4j/main/core/src/main/java/com/ibm/icu/text/Normalizer.java#L233-L236

see https://github.com/search?q=repo%3Aunicode-org%2Ficu%20language%3Ajava%20%22%40internal%22&type=code for other examples

FYI @yumaoka

Reply:
Yeah, you're right.  I just got lazy.  Fixed now.